pep-9999.rst

```diff
@@ -559,6 +559,17 @@ No::
         ...
     class AnotherError(Exception): pass
 
+Use the type annotation syntax for instance attributes. Class attributes
+must be annotated with ``typing.ClassVar``.
```

Comment:
I'm actually not sure this is a "must". Most of typeshed doesn't do it (in part because it predates the PEP that added ClassVar), and type checking still mostly works if you get it wrong. I believe the only thing that ClassVar would do in normal classes is to set `instance.universal_thing = "something"`.

Reply:
You are right, sorry for the confusion.